### ISSUE TITLE
UUID field not serializable in middleware

### ISSUE THREAD

**Issue by OllyPage** [vishalanandl177/DRF-API-Logger#45]
`TypeError: Object of type UUID is not JSON serializable`

Specifically here: `d['headers'] = json.dumps(d['headers'], indent=4)`

The field is of type uuid.UUID .

I have found using the DjangoJSONEncoder allows for the encoding of these odd django fields
`d['headers'] = json.dumps(d['headers'], indent=4, cls=DjangoJSONEncoder)`

**Comment by vishalanandl177** [comment]
Can you please share the payload here.

**Comment by OllyPage** [comment]
This is what is being produced from a test. Looking into it it could be that django tests allow http content to contain types that are not strings but a real client header would be of type string not UUID.
I haven't investigated but it might be the case that a response containing a type that cannot be serialized by json.dumps (e.g. UUID) is returned by an endpoint and could lead to this issue in production. I don't have any endpoints that return that return UUIDs so this isn't something I've tested. I can test this at some point if you think that could be the case.

Here is the header from the test case that sets CLIENT_ID=model.id:UUID
Header: `{'COOKIE': '***FILTERED***', 'CLIENT_ID': UUID('2dd57000-37e4-41cf-8f50-cd9bf6664000')}`
A deployment header would look like 
Header: `{'COOKIE': '***FILTERED***', 'CLIENT_ID': '2dd57000-37e4-41cf-8f50-cd9bf6664000'}`

BTW I've found your library very helpful, cheers!
